bokehjs/src/lib/models/tools/inspectors/crosshair_tool.ts

```diff
@@ -76,12 +76,26 @@ export class CrosshairToolView extends InspectToolView {
     }
 
     const {sx, sy} = ev
-
-    if (!this.plot_view.frame.bbox.contains(sx, sy)) {
-      this._update_spans(NaN, NaN)
-    } else {
+    if (this.plot_view.frame.bbox.contains(sx, sy)) {
       this._update_spans(sx, sy)
+      return
```

Comment:
Usually fine with early returns but I like them less when there's not an actual value. I guess the logic could be pulled into a function that returns `[sx, sy]` then just have one call to `this._update_spans(sx, sy)` after.